sw/host/provisioning/orchestrator/src/orchestrator.py

```diff
@@ -92,7 +92,6 @@ def main(args_in):
     )
     parser.add_argument(
         "--rma-unlock-token",
-        required=True,
```

Comment:
We should probably make this parameter optional in the orchestrator.py script too: https://cs.opensource.google/opentitan/opentitan/+/earlgrey_1.0.0:sw/host/provisioning/orchestrator/src/ot_dut.py;l=166